modules/rosapps/applications/sysutils/utils/rosperf/rosperf.c

```diff
@@ -290,12 +290,11 @@ ProcessTest(PTEST Test, PPERF_INFO PerfInfo)
 static void
 PrintOSVersion(void)
 {
-#define BUFSIZE 160
   OSVERSIONINFOEXW VersionInfo;
   BOOL OsVersionInfoEx;
   HKEY hKey;
-  WCHAR ProductType[BUFSIZE];
-  DWORD BufLen;
+  WCHAR ProductType[9] = { L'\0' };
```

Comment:
Why not `= L"";` ?

Reply:
Because all the characters must be initialized: `L""` would help empty string case only

Reply:
The two behave identically. There is no partial initialization in C, any initialization that doesn't specify some array/struct members will set the rest to zero.